assets/js/admwpp-debug.js

```diff
@@ -1,8 +1,148 @@
 (function ($) {
+  $.ADMMessage = function () {
 
-  $.ADMShortcode = function () {
+    var self = this;
+
+    // Init logic
+    var defaults = $.ADMMessage.defaults;
+
+    // Add confirm dialog html.
+    if ($(defaults.confirm_dialog).length == 0) {
+      var html = "<div id='admwpp-dialog-confirm' class='admwpp-dialog-confirm'>";
+      html += "<p><span id='admwpp-confirm-dialog-message'></span></p>";
+      html += "</div>";
```

Comment:
can we use the string literals ` to instead of concatenating or maybe write it in a one line

Reply:
i don't see the point of doing so can you elaborate on the advantages? I'm breaking the html into smaller peaces not to have it in a single line for readabilty.